scripts/testing/rebuild_boot_iso

```diff
@@ -50,6 +54,15 @@ rm -f ${BOOT_ISO_GIT_REVISION}
 mkdir -p result/iso/
 # note the Git revision from which we build the boot.iso
 git rev-parse HEAD > result/iso/boot.iso.git_rev
+
+# build the anaconda rpms
 make -f ./Makefile.am container-rpms-scratch
+
+# copy additional web UI packages
```

Comment:
Couldn't this be in principle any RPMs ?

Reply:
yes it will take any RPM

Reply:
Then I suggest changing the single line comment - `"# copy additional RPM packages"`